projects/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-annotatitons/dataset-annotations.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import produce from 'immer';
+import { merge } from 'lodash-es';
+import { DataLayerColorService } from '../../data-layer/data-layer-color.service';
+
+@Component({
+  selector: 'dataset-annotations',
+  templateUrl: './dataset-annotations.component.html',
+  styleUrls: ['./dataset-annotations.component.css'],
+})
+export class DatasetAnnotationsComponent implements OnInit {
```

Comment:
Please rename this component to `AnnotationOptionsComponent` (including selector and filenames)